package.json

```diff
@@ -26,7 +26,7 @@
     },
     "homepage": "https://github.com/prototypsthlm/contentful-migrator-programme#readme",
     "dependencies": {
-        "chalk": "^5.0.1",
```

Comment:
https://github.com/chalk/chalk/releases/tag/v5.0.0

Version 5 would make us convert the project to ESM, so I'm decreasing this version